lib/swift/Sources/LinuxHelper.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  LinuxHelper.swift
+//  Thrift
+//
+//  Created by Christopher Simpson on 8/22/16.
+//
+//
+
+import Foundation
+import CoreFoundation
+
+#if os(Linux)
+/// Extensions for Linux for incomplete Foundation API's.
+/// swift-corelibs-foundation is not yet 1:1 with OSX/iOS Foundation
+
+extension URLSession {
+  // Current one uses NSURLRequest which doesn't currently bridge
+  @discardableResult
+  open func dataTask(with request: URLRequest, completionHandler: @escaping (Data?, URLResponse?, Error?) -> Void) -> URLSessionDataTask {
```

Comment:
@fumoboy007 Which tag was that added in?  I'm attempting to have this match up with GM (its been tricky the past few weeks -_-)
I don't mind floating it since on the Darwin side, this stuff is all good to go where its hard to update swift, whereas the linux side its a little easier to update Swift to snapshots to ensure a solid library build.